rct229/rules/section5/section5rule48.py

```diff
@@ -0,0 +1,79 @@
+from rct229.rule_engine.rule_base import (
+    RuleDefinitionBase,
+    RuleDefinitionListIndexedBase,
+)
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.get_zone_conditioning_category_dict import (
+    get_zone_conditioning_category_dict,
+)
+from rct229.utils.jsonpath_utils import find_all
+
+
+class Section5Rule48(RuleDefinitionListIndexedBase):
+    """Rule 48 of ASHRAE 90.1-2019 Appendix G Section 5 (Envelope)"""
+
+    def __init__(self):
+        super(Section5Rule48, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, True),
+            required_fields={
+                "$": ["weather"],
+                "weather": ["climate_zone"],
+            },
+            each_rule=Section5Rule48.BuildingRule(),
+            index_rmr="baseline",
+            id="5-48",
+            description="The air leakage rate in unconditioned and unenclosed spaces must be the same the baseline and proposed design.",
+            list_path="ruleset_model_instances[0].buildings[*]",
+        )
+
+    def create_data(self, context, data=None):  # put it under the "BuildingRule"
+        rmr_baseline = context.baseline
+        return {"climate_zone": rmr_baseline["weather"]["climate_zone"]}
+
+    class BuildingRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section5Rule48.BuildingRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, True),
+                required_fields={
+                    "$..zones[*]": ["infiltration"],
+                    "$..infiltration[*]": ["infiltration_flow_rate"],
+                },
+            )
+
```

Comment:
Since the RDS assertion applies to each zone, I think we need to use a nested class `ZoneRule` inside of `BuildingRule`.
One advantage of using a nested `ZoneRule` is that the zones will automatically be matched up.